src/site/xdoc/server/config-smtp-lmtp.xml

```diff
@@ -151,8 +151,8 @@
       
         <li>As mentioned above, SMTP AUTH requires that Apache James Server be able to distinguish between mail intended 
             for local delivery and mail intended for remote delivery.  Apache James Server makes this determination by matching the 
-            domain to which the mail was sent against the &lt;servernames&gt; element of the Apache James Server configuration block.  Any 
-            local domains should be explicitly listed as &lt;servername&gt; elements in this section.</li>
+            domain to which the mail was sent against the &lt;domainnames&gt; element of the Apache James Server configuration block.  Any 
```

Comment:
You are right to show us this configuration is out-dated.

However, to be correct it should be:

```
to which the mail was sent against the &lt;DomainList&gt; component, configured in <a href="/server/config-domainlist.xml">domainlist.xml</a> configuration file.
```

Cheers

Reply:
Great, I'll adjust it. thank you very much.

Reply:
done. please review .